WorldModel/WorldModel/Core/CoreOutput.aslx

```diff
@@ -769,6 +769,16 @@
       }
     }
   ]]></function>
+
+  <function name="ProcessTextCommand_Element" parameters="section, data" type="string"><![CDATA[
+    if (IsRegexMatch(":?/?\\w+\\b((?<=\\s)\\w+\\b|(?<=\\s)\\w+=\\w+\\b|(?<=\\s)\\w+=\"[^\"]*\"|(?<=\\s)\\w+='[^']*')*/?\\s*", section)) {

```

Comment:
That's... quite the Regex. Where did that come from and what is it supposed to do?

Reply:
> I'm a bit dubious of this one, if it's just for WebEditor users who are facing some bug when they use `<`, we should just fix that instead.

That's all it's for. I thought not being able to use `<` in the WebEditor was a known issue with no workaround. If that can be fixed, that sounds much better.

---
> That's... quite the Regex. Where did that come from and what is it supposed to do?

Haha... It works. have no clue *how* it works, but it handled everything Pertex and I could think of to test it.

It came from mrangel in the forums: https://textadventures.co.uk/forum/quest/topic/ykouw7idaeytupcl3rmywa/isnt-there-an-easy-way-to-add-text-processor-commands-solved